tests/unit/r2i/tflite/resources/meson.build

```diff
@@ -0,0 +1,9 @@
+tflite_tests_resources = [
+  ['squeezenet.tflite'],
+  ['invalid.tflite'],
+]
+
+# Copy resources files
+foreach t : tflite_tests_resources
+  configure_file(copy : true, input : t[0], output : t[0])
```

Comment:
This feels unneeded. Consider other options in the meson.build one level above.

Reply:
What other options? copy the entire directory from the level above is not possible (or I didn't find a way to do so).

And as mentioned above I prefer to copy the resources to build dir